packages/util/src/api.ts

```diff
@@ -1569,6 +1569,76 @@ export class Api extends BaseApi {
       : secondAmountNum.div(!firstAmountNum.isZero() ? firstAmountNum : one)
     return result.format()
   }
+
+  /**
+   * Returns the referral of the invited user by Id
+   * @param invitedUserId address of invited account
+   * @returns referral
+   */
+  public async getReferral (invitedUserId: string): Promise<string> {
+    const referral = await this.api.query.referrals.referrers(invitedUserId) as any
+    return !referral ? '' : referral.toString()
+  }
+
+  /**
+   * Returns invited users of the referral
+   * @param referralId address of referral account
+   * @returns array of invited users
+   */
+  public async getInvitedUsers (referralId: string): Promise<Array<string>> {
+    const referrals = await this.api.query.referrals.referrals(referralId) as any
+    if (!referrals) {
+      return []
+    }
+    return referrals
+  }
+
+  /**
+   * Transfer XOR balance from the referral account to the special account
+   * This balance can be used by referrals to pay the fee
+   * @param amount balance to reserve
+   */
+  public async reserveXor (amount: NumberLike): Promise<void> {
+    assert(this.account, Messages.connectWallet)
+    const asset = await this.getAssetInfo(KnownAssets.get(KnownSymbols.XOR).address)
+    const assetAddress: string = asset?.address
```

Comment:
Do we need the call above, if we can just use `KnownAssets.get(KnownSymbols.XOR)` or (`XOR` exported from assets.ts) data?